vernac/comProgramFixpoint.ml

```diff
@@ -262,7 +262,8 @@ let build_wellfounded (recname,pl,bl,arityc,body) poly r measure notation =
   let uctx = Evd.evar_universe_context sigma in
   let cinfo = Declare.CInfo.make ~name:recname ~typ:evars_typ () in
   let info = Declare.Info.make ~udecl ~poly ~hook () in
-  let _ : Declare.progress = Declare.Obls.add_definition ~cinfo ~info ~term:evars_def ~uctx evars in
+  let _ : Declare.Obls.progress =
+    Declare.Obls.add_definition ~cinfo ~info ~term:evars_def ~uctx evars in
```

Comment:
Since the `progress` is always ignored outside declare maybe we should just return unit.
I'll push a commit so we can see what it looks like.

Reply:
Ups, if you pushed I'm afraid I overwrote it, I missed the comment sorry. Let me try to recover it, but I think `progress` is used in equations.

[the CI failed and I was puzzled why, indeed your commit should be the reason]

Reply:
Equations uses this, and indeed it seems like valuable info in the general API, so I'm leaving it.